js/ui/html_editor/modules/toolbar.js

```diff
@@ -540,7 +236,7 @@ if(Quill) {
                     hint: localize(buttonText),
                     text: localize(buttonText),
                     icon: iconName.toLowerCase(),
-                    onClick: this._formatHandlers[name] || this._getDefaultClickHandler(name),
+                    onClick: this._formatHandlers[name] || getDefaultClickHandler(name, this),
```

Comment:
```suggestion
                    onClick: this._formatHandlers[name] || getDefaultClickHandler(this, name),
```
`context` then `handler name`
like [Function.prototype.apply()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply) `.apply(context, arg1, arg2, ...)`

Check other exports from toolbar_helper, pls. They're should look consistent with the others